scripts/windows/build/Publish-DockerImageSuite.ps1

```diff
@@ -25,7 +25,7 @@ param (
     [Switch]$Agent,
     [Switch]$Hub,
     [Switch]$SimulatedTemperatureSensor,
-    [Switch]$FunctionsBinding,
+    [Switch]$TemperatureFilter,
```

Comment:
I don't see any usage in this file; therefore I remove it.  Understood that we have function binding test, will add it later if needed.